classes/Product.php

```diff
@@ -5466,6 +5466,7 @@ public static function getProductProperties($id_lang, $row, Context $context = n
         $row['category'] = Category::getLinkRewrite((int) $row['id_category_default'], (int) $id_lang);
         $row['category_name'] = Db::getInstance()->getValue('SELECT name FROM ' . _DB_PREFIX_ . 'category_lang WHERE id_shop = ' . (int) $context->shop->id . ' AND id_lang = ' . (int) $id_lang . ' AND id_category = ' . (int) $row['id_category_default']);
         $row['link'] = $context->link->getProductLink((int) $row['id_product'], $row['link_rewrite'], $row['category'], $row['ean13']);
+        $row['manufacturer_name'] = Manufacturer::getNameById((int) $row['id_manufacturer']);
```

Comment:
```suggestion
        $row['manufacturer_name'] = (int) $row['id_manufacturer'] > 0 ? Manufacturer::getNameById((int) $row['id_manufacturer']) : null;
```

Other option is to change the `Manufacturer::getNameById` to prevent checking if the ID is not > 0

One SQL query less for the product if we don't need it. What do you think, @PululuK ?

Reply:
Yeah !